frontend/src/pages/CourseSelector/CourseSelector.jsx

```diff
@@ -16,7 +16,7 @@ const CourseSelector = () => {
   const dispatch = useDispatch();
 
```

Comment:
in this file, i was also thinking that we should find a way to sort the containers - possibly by level? not sure.